### ISSUE TITLE
Air trapping

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6685]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Air trapping is retention of air in the lung distal to an obstruction (usually partial). Air trapping is seen on end-expiration CT scans as parenchymal areas with less than normal increase in attenuation and lack of volume reduction. Comparison between inspiratory and expiratory CT scans can be helpful when air trapping is subtle or diffuse.

PMID:18195376

Comment
Differentiation from areas of decreased attenuation resulting from hypoperfusion as a consequence of an occlusive vascular disorder (eg, chronic thromboembolism) may be problematic, but other findings of airways versus vascular disease are usually present. See Figure 4 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
